dataactbroker/handlers/fileHandler.py

```diff
@@ -562,7 +567,8 @@ def checkGeneration(self, submission_id = None, file_type = None):
         if uploadJob.filename is None:
             responseDict["url"] = ""
         elif CONFIG_BROKER["use_aws"]:
-            responseDict["url"] = s3UrlHandler().getSignedUrl("d-files",uploadJob.filename, bucketRoute=None, method="GET")
+            path, file_name = uploadJob.filename.split("/")
```

Comment:
I didn't think we'd need it since the download is only coming from the user's directory on the specified S3 bucket. If a user ever gets nested directories, I would agree, but we should check if there will ever be such a scenario.